streamd/prolif/prolif_md.py

```diff
@@ -56,56 +34,26 @@ def main():
                         help='output text file name')
     parser.add_argument('-v', '--verbose', action='store_true', default=False,
                         help='print progress.')
-    parser.add_argument('-c', default=1, type=int,
-                        help='ncpu')
-#   parser.add_argument('--renum_chain', action='store_true', default=False)
-
 
     args = parser.parse_args()
     u = mda.Universe(args.topology, args.trajectory)
```

Comment:
these arguments are absent in the command line options, there were removed
`args.topology, args.trajectory`